src/app/api/actions/modifierUneNoteDeContexteAction.ts

```diff
@@ -0,0 +1,8 @@
+'use server'
+
+import { revalidatePath } from 'next/cache'
+
+export async function modifierUneNoteDeContexteAction(): Promise<Array<string>> {
```

Comment:
```suggestion
export async function modifierUneNoteDeContexteAction(): Promise<ReadonlyArray<string>> {
```

Pourquoi un tableau ?